commit 2714eb4523b74e068052883181c987a0a60753e3
Author: John Levon <john.levon@joyent.com>
Date:   2019-07-08T09:53:22+00:00 (3 months ago)
    
    joyent/node-zkstream#9 use var instead of const so we can run in strict mode
    joyent/node-zkstream#29 unreliable tests assuming after sync() all watchers have fired
    joyent/node-zkstream#24 remove watchers from index once all event handlers are gone

diff --git a/README.md b/README.md
index ca63117..30c4a3d 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 zkstream
 ========
 
+
 A minimal streams-based node client for the ZooKeeper protocol.
 
 API
